### ISSUE TITLE
utf-8 support

### ISSUE THREAD

**Issue by fevxie** [TruckMovers/cordova-plugin-remote-injection#13]
Hi:

   I use this plugin inject a javascript file on the remote website, it worked perfect. But when I write  some none ascii chars in the javascript file, on the Android platform, I found the none ascii chars display wrong. I think maybe the problem should be settled on this plugin?  

the javascript like :

```
navigator.notification.alert(
                message,    // message
                null,       // callback
                "", // title
                '确定'        // buttonName
            );
```

I use none ascii code on the button name,  please give me some advices. 

Thanks.

**Comment by bradleyjames** [comment]
Is this a JS file that you're providing?  If so try setting the file's encoding to UTF-8 as described in this [SO post](http://stackoverflow.com/a/32912667/214675).  Let me know what happens.

**Comment by fevxie** [comment]
hi, @bradleyjames :
 
yes, this is my own js file using to reject into remote website. Trying the method you suggested, but It doesn't work. 

PS: the issues only exist on android platform,  iOS platform doesn't have this issues.

Attach file: the result of alter output when using unicode char.

![layout](https://cloud.githubusercontent.com/assets/2520264/18497461/f73c9438-7a60-11e6-865f-638e05e9bde9.png)

**Comment by bradleyjames** [comment]
I was able to reproduce.  I'll try to figure out what's going on.

**Comment by bradleyjames** [comment]
Thanks for the PR!  It's been merged and I released v0.4.1 to npm.
